.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,32 @@
+**Note: for support questions, please use stackoverflow**.
+This repository's issues are reserved for feature requests and bug reports.
+
+* **Type of issue**
```

Comment:
This section doesn't seem applicable to a pull request. Maybe a field like "related issue (if applicable)".

Category might make more sense, with options being "bug fix", "new feature", and "other (describe)"